src/reader/reader.ts

```diff
@@ -47,24 +49,153 @@ export function putContentIntoLines(contents: string): Query[] {
 }
 
 /**
- * 1. Split on new line
- * 2. Filter out any lines that start with a comment
- * 3. Rejoin the lines together as a single string.
+ * Function to escape any special characters in a string to allow
+ * it to be passed to build a regex.
+ * Based on https://stackoverflow.com/a/32212181/2144578
+ */
+function regexEscape(str: string): string {
+  str = str.replace(/\\/g, "\\\\");
+  str = str.replace(/\-/g, "\\-");
+  str = str.replace(/\//g, "\\/");
+  str = str.replace(/\./g, "\\.");
+  str = str.replace(/\^/g, "\\^");
+  str = str.replace(/\$/g, "\\$");
+  str = str.replace(/\*/g, "\\*");
+  str = str.replace(/\+/g, "\\+");
+  str = str.replace(/\-/g, "\\-");
+  str = str.replace(/\?/g, "\\?");
+  str = str.replace(/\(/g, "\\(");
+  str = str.replace(/\)/g, "\\(");
+  str = str.replace(/\[/g, "\\[");
+  str = str.replace(/\]/g, "\\]");
+  str = str.replace(/\{/g, "\\{");
+  str = str.replace(/\}/g, "\\}");
+  str = str.replace(/\|/g, "\\|");
+  return str;
+}
+
+/**
+ * Strips content between two delimeters, starting with a regex
+ * Works across multiple lines (hence the name!) but preserves the newlines
+ * themselves to allow error reporting to reflect source lines.
+ *
+ * Note this requires dotAll regex (s) to work across multiple lines so only
+ * really works on node 9+.
```

Comment:
As a note, the node9+ version constraint is fine. At the moment I'm targeting 12+ but I plan on moving to 14+ when node14 enters the 'active' status

Reply:
Cool. Will remove the note.